strings/aho_corasick.py

```diff
@@ -62,21 +71,28 @@ def set_fail_transitions(self) -> None:
                     + self.adlist[self.adlist[child]["fail_state"]]["output"]
                 )
 
-    def search_in(self, string: str) -> Dict[str, List[int]]:
+    def search_in(self, string: str) -> dict[str, list[int]]:
         """
         >>> A = Automaton(["what", "hat", "ver", "er"])
         >>> A.search_in("whatever, err ... , wherever")
         {'what': [0], 'hat': [1], 'ver': [5, 25], 'er': [6, 10, 22, 26]}
         """
-        result = dict()  # returns a dict with keywords and list of its occurrences
+        result: dict[
+            str, list[int]
+        ] = dict()  # returns a dict with keywords and list of its occurrences
```

Comment:
Can you put the comment above the assignment statement and reformat the code?

This way the line will be consistent and easy to look.

Reply:
I can move the comment but reformatting is done by black so can't change that